great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -367,21 +373,26 @@ def get_batch_data_and_markers(  # noqa: C901, PLR0912, PLR0915
 
         return typed_batch_data, batch_markers
 
-    def _apply_splitting_and_sampling_methods(self, batch_spec, batch_data):
-        splitter_method_name: Optional[str] = batch_spec.get("splitter_method")
-        if splitter_method_name:
-            splitter_fn: Callable = self._data_splitter.get_splitter_method(
-                splitter_method_name
-            )
-            splitter_kwargs: dict = batch_spec.get("splitter_kwargs") or {}
-            batch_data = splitter_fn(batch_data, **splitter_kwargs)
+    def _apply_splitting_and_sampling_methods(
+        self,
+        batch_spec: BatchSpec | PandasBatchSpecProtocol,
+        batch_data: PandasBatchData,
+    ):
+        if isinstance(batch_spec, BatchSpec):
+            splitter_method_name: Optional[str] = batch_spec.get("splitter_method")
```

Comment:
Splitting and sampling has not been implemented. Let's skip this for the first iteration.
Also, this isn't the best way to deal with this issue.

Reply:
I would say it's more like partitioning the data based on some category, like directories, or a date-range.